roles/kubespray-defaults/defaults/main/checksums.yml

```diff
@@ -785,23 +785,34 @@ crun_checksums:
     1.9.2: sha256:42813b5bea2137bf9abcd1bcaa098a7d61fbbffd2a35d9c9f0f1ba79fb74eb5b
 youki_checksums:
   amd64:
-    0.4.1: sha256:6504a43c28710d2cb3dd6535ae9222c0570e954a799a787ccf5e5d611996bf11
-    0.4.0: sha256:a9fb31c7388ed786a2a1b6361cf4aaa7c3e3b62be4c3d36dc15331416a3d6290
-    0.3.3: sha256:f6e091ad373cfaefcb222f709ef387834eb679c2dedc27a2e0ca60052f31edba
-    0.3.2: sha256:fe80475ad6e727a50f7c0b89e26e4632dd55e477c99bbca192ad1d5fee065377
-    0.3.1: sha256:8a92304312982246d5ad9c00da367868dcc4d331fd992ca89dada8eff9cbdeaf
-    0.3.0: sha256:741ba3cd85d768bebba02598cedcf3b15a2160e4d6ce33a3d5c4e1b3080f9c1c
-    0.2.0: sha256:b268689a91db07feebfd41d5806b10c7d051fbcbf7efb15076e2228763ac0762
-    0.1.0: sha256:f00677e9674215b44f140f0c0f4b79b0001c72c073d2c5bb514b7a9dcb13bdbc
+    0.5.3: sha256:173b8998cd0abf22e38e36611b34cc19a16431b353dd893e3d988cfc77b4e6ac
+    0.5.2: sha256:361c7187939eab02039fa4289d33158fe4bf4d21ab8cc7139ace8f52081524ba
+    0.5.1: sha256:554dea487dcb54b34fff003fc047458cecf073337d20b77e69c22918c2986aaf
+    0.5.0: sha256:d7ec0cd5f69ecd96b41df13cc38a970ab217aa9c4b2262ca4843a68595975d91
+    0.4.1: sha256:6f7ea3651b284a808f344ad40f9f8315b3b02b76a3ee6c6af7bfff65753284c9
```

Comment:
Why `0.4.1` & `0.4.0` hash changed?

Reply:
It seems difference between 'gnu' and 'musl' versions.
```
$ sha256sum youki-0.4.1-x86_64-*.gz
6f7ea3651b284a808f344ad40f9f8315b3b02b76a3ee6c6af7bfff65753284c9  youki-0.4.1-x86_64-gnu.tar.gz
6504a43c28710d2cb3dd6535ae9222c0570e954a799a787ccf5e5d611996bf11  youki-0.4.1-x86_64-musl.tar.gz
```

Reply:
Does it sound like the hash was wrong in the first place?

Reply:
@tico88612 😀
The original hash was correct (musl-based). 
This update switches youki to a gnu-based build, hence the hash change.

Reply:
Got it, thanks!

/lgtm